exercises/armstrong-numbers/src/main/java/ArmstrongNumbers.java

```diff
@@ -0,0 +1,9 @@
+class ArmstrongNumbers {
+
+	boolean isArmstrongNumber(int i) {
```

Comment:
Sorry, just one thing I missed last time; `i` is normally only used as a variable name for an index. Maybe `numberToCheck`, `possibleArmstrongNumber` or `inputNumber` or somthing might be more descriptive? :)